js-intro-homework/js-excercise-1/js-excercise-1.js

```diff
@@ -0,0 +1,17 @@
+// a)
+let contry = 'Macedonia';
```

Comment:
This is fine, but in this case you can use "const" as well, because we know that the country variable won't change during the code execution